src/pocketmine/tile/Sign.php

```diff
@@ -55,11 +55,19 @@ public function saveNBT(){
 		unset($this->namedtag->Creator);
 	}
 
-	public function setText($line1 = "", $line2 = "", $line3 = "", $line4 = ""){
```

Comment:
Please also add PhpDoc to this function to make sure it's clear, since there are no typehints.